code/modules/shuttle/special.dm

```diff
@@ -378,15 +378,14 @@
 		var/change = FALSE
 		if(payees[AM] > 0)
 			change = TRUE
-			var/obj/item/holochip/HC = new /obj/item/holochip(AM.loc) //Change is made in holocredits exclusively.
-			HC.credits = payees[AM]
-			HC.name = "[HC.credits] credit holochip"
+			var/obj/item/holochip/holocred = new /obj/item/holochip(AM.loc, payees[AM]) //Change is made in holocredits exclusively.
+			holocred.name = "[holocred.credits] credit holochip"
```

Comment:
```suggestion
```

we do this in `update_name()` invoked by `Initialize()` on the holochip, no need to do this extra work so lets get rid of it while we're in the area.